.github/workflows/code-quality.yml

```
@@ -328,53 +369,56 @@ jobs:
       - uses: astral-sh/setup-uv@d4b2f3b6ecc6e67c4457f6d3e41ec42d3d0fcb86 # v5.4.2
         with:
           version: "0.8.17"
           enable-cache: true
           prune-cache: true
-      - name: ruff check + format (PL, RUF, FAST, S, B, A, COM, C4, DTZ, ISC, RET, SIM, ARG, PTH, TRY, PERF, UP, FURB, LOG, G + more)
-        # Pinned to the uv.lock ruff version — bump both together. Unpinned,
-        # a ruff release can flip this lane red with zero code changes
-        # (0.16.0 shipped ISC004/RUF036 four minutes before a run here).
+      - uses: ./.github/actions/setup-node-pnpm
+      - name: ruff via nx affected (cache:true)
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:4:208: Double quote to prevent globbing and word splitting

(shellcheck)